bluetooth_mesh/models/models.py

```diff
@@ -1468,10 +1468,10 @@ class LightLightnessClient(Model):
     SUBSCRIBE = True
     _tid = 0
 
-    @property
     def tid(self) -> int:
```

Comment:
Why not `self.__tid = itertools.cycle(range(255))` in ctor and then just `tid = next(self.__tid)`, like in the scene client?

Reply:
Turning property into method introduces less changes. Of course I could use itertools.cycle, but isn't this simpler?

Reply:
This version is simpler and changes less code.

Reply:
Ok, I'm fine either way. Just please make it consistent with other models